samples/loadimpactair_script_sample.js

```diff
@@ -0,0 +1,397 @@
+﻿// This script is a user journey for http://loadimpactair.guldskeden.se
+
+// LOAD IMPACT AB
+// 
+// Copyright 2017 Load Impact AB
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.See the License for the specific language governing permissions and limitations under the License.
+//
+
+import { group, sleep } from "k6";
+import http from "k6/http";
+import { Trend } from "k6/metrics";
+
+/**
+ * Returns random number between (inclusive) min and max
+ */
+function rnd(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+/**
+ * Returns URL encoded string
+ */
+function urlencode (str) {

```

Comment:
It's less work to split a form string into an object than to write a function to correctly handle URL encoding in JS…?